vault/views/vault_views.xml

```diff
@@ -43,16 +43,19 @@
                         type="object"
                         name="action_open_import_wizard"
                         string="Import from file"
+                        groups="vault.group_vault_import"
                     />
                     <button
                         type="object"
                         name="action_open_export_wizard"
                         string="Export to file"
+                        groups="vault.group_vault_export"
                     />
                     <button
                         type="object"
                         name="action_open_export_wizard"
                         string="Export to file"
+                        groups="vault.group_vault_export"
                     />
```

Comment:
I think button is duplicated and should be removed, right?

Reply:
You are right, changed